app/src/sharedTest/java/org/oppia/app/administratorcontrols/AppVersionActivityTest.kt

```diff
@@ -79,7 +78,7 @@ class AppVersionActivityTest {
         withText(
           String.format(
             context.resources.getString(R.string.app_version_name),
-            BuildConfig.VERSION_NAME
+            context.packageManager.getPackageInfo(context.packageName, 0).versionName
```

Comment:
Maybe should move this to an extension function on package manager so that it can be reused everywhere that we need it. What do you think?

Reply:
Do you mean create a function that would replace `.getPackageInfo(context.packageName, 0).versionName` ?

Reply:
Yes, something like:

```kotlin
fun PackageManager.getVersionName(): String {
  return getPackageInfo(...).versionName
}
```

Then we can do ``context.packageManager.getVersionName()`` where needed.

Reply:
Sure!